public/test/examples/doc_rr_objects.html

```diff
@@ -33,7 +33,7 @@
   g[`a${i}`] = i;
 }
 var h = /abc/gi;
-var i = new Date();
+var i = new Date(1666130037374);
```

Comment:
I'm curious about this change?

Reply:
The old fixture was a pain because the date value changed every time we re-recorded the example! This way the test fixture always produces the same values. See https://github.com/replayio/devtools/pull/7971/files#diff-b30a7a48057849c2194202ed696d2d9e394ab383acb8d5468c2b223d647bc6a4R28

Reply:
That's fair! 😄 If